spec/support/it_accepts_metadata_json_targets.rb

```diff
@@ -17,28 +17,29 @@
         let(:globbed_files) { [module_metadata_json] }
 
         it 'returns the path to metadata.json in the module' do
-          expect(parsed_targets).to eq(globbed_files)
+          expect(parsed_targets.first).to eq(globbed_files)
         end
       end
 
       context 'and the module does not contain a metadata.json file' do
         it 'returns no targets' do
-          expect(parsed_targets).to eq([])
+          expect(parsed_targets.first).to eq([])
         end
       end
     end
 
-    context 'when given specific target files' do
+    context 'when given unmatched target files' do
```

Comment:
Not sure I'd understand this immediately when reading.

Perhaps

    context 'when given unmatching target files' do

or

    context 'when given a target that will not match the validator\s pattern' do

?